.github/workflows/run_fuzzer.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         buildtype: 'boost'
         path: 'head'
         toolset: clang-18
+        cxxstd: 14
```

Comment:
Why have you reduced this from 17 to 14?

Reply:
I wanted to make sure this MR has a chance to be approved and avoid potential OOMs on the drone agent before (although I'm not sure if it's related to the current MR). I reverted to 17; please let me know if there are any issues with it.